### PR TITLE
Properties should not perform api requests

### PR DESCRIPTION
**Pull request by whyscream**

### CODE REVIEW

File: mollie/api/resources/captures.py [hunk 1]
```diff
@@ -36,5 +36,5 @@ def __init__(self, client, settlement):
         self._settlement = settlement
         super().__init__(client)
 
-    def get_resource_path(self):
-        return f"settlements/{self._settlement.id}/captures"
+    def get_resource_path(self) -> str:
+        return f"settlements/{self._settlement.id}/captures"  # type:ignore
```
Comment (geertjanvdenbosch) [inline mollie/api/resources/captures.py:40]: Why use `# type: ignore` here?

Comment (whyscream) [reply mollie/api/resources/captures.py:40]: The `self._settlement` property is typed as `None`, so it cannot have an `.id` property. 

Some older python versions (tested with 3.7) cannot parse proper typing for `self._settlement` so we can't add that (yet). Ignoring the message about incorrect typing on `self._settlement` here is about the best we can do.

File: mollie/api/objects/capture.py [hunk 2]
```diff
@@ -40,14 +40,12 @@ def settlement_id(self):
     def created_at(self):
         return self._get_property("createdAt")
 
-    @property
-    def payment(self):
+    def get_payment(self):
         """Return the payment for this capture."""
-        # TODO Use the embedded payment data, if avalable.
+        # TODO Use the embedded payment data, if available.
```
Comment (geertjanvdenbosch) [inline mollie/api/objects/capture.py:45]: Is this available yet?

Comment (whyscream) [reply mollie/api/objects/capture.py:45]: It depends on whether the developer requested embedded payment data in the capture request: https://docs.mollie.com/reference/v2/captures-api/get-capture#embedding-of-related-resources

**Review by geertjanvdenbosch — APPROVED** [review]

**Pull request merged**